pkg/m3admin/health/client.go

```diff
@@ -30,8 +32,8 @@ import (
 )
 
 const (
-	bootstrappedPath = "/bootstrapped"
-	m3dbServiceName  = "m3dbnode-m3-cluster-short"
+	healthPath      = "/health"
```

Comment:
Wouldn't it make more sense to just add this as a check on the client in addition to `/bootstrapped`?

Reply:
If not, I'd think you'd want to rename client method from `Bootstrapped` to something like `Health`

Reply:
i don't totally follow. but the problem is this basically would never return false. it would only return true or error.

Reply:
More so suggesting to make it consistent with the API we're calling.  Ideally:
```
HealthClient {
  Bootstrapped() error  // Or just delete this
  Health() (HealthResult, error)
}
```

As it stands, we've got the `Bootstrapped` method invoking the `/health` endpoint which seems inconsistent.